src/mixins/nft.js

```diff
@@ -147,15 +147,18 @@ export default {
       };
     },
     listingInfo() {
-      const list = this.getNFTClassListingInfoById(this.classId) || {};
-      const { price, nftId, seller } = list;
-      return {
-        price,
-        totalPrice: price,
-        classId: this.classId,
-        nftId,
-        seller,
-      };
+      const list = this.getNFTClassListingInfoById(this.classId) || [];
+      if (list.length) {
+        const { price, nftId, seller } = list[0];
+        return {
+          price,
+          totalPrice: price,
+          classId: this.classId,
+          nftId,
+          seller,
+        };
+      }
+      return undefined;
```

Comment:
I think the old code is actually bad (it returns `{}` instead of `undefined` if there is no listing)
But returning `undefined` now would be breaking change, so we better return `{}` for now